src/main/java/org/jabref/logic/integrity/FieldCheckers.java

```diff
@@ -49,6 +50,10 @@ public FieldCheckers(BibDatabaseContext databaseContext, FilePreferences filePre
         fieldCheckers.put(FieldName.URL, new UrlChecker());
         fieldCheckers.put(FieldName.YEAR, new YearChecker());
         fieldCheckers.put(FieldName.KEY, new ValidBibtexKeyChecker(enforceLegalKey));
+        // Fix #1 imported org.jabref.model.entry.BibEntry and added the BibEtry.KEY_FIELD
```

Comment:
There is a small problem in that the key checker for the `key`(= sort key) now also complains about an empty key, although this field is optional. Probably the easiest solution is to add a boolean toggle for the empty check.

Reply:
I'm sorry, I'm not 100% sure what you mean here. Would it be possible to send a screenshot of the error? If it is on the preview panel, seeing exactly the tab, the field, and what exactly it is doing would help me understand and pinpoint the error.

Reply:
Sorry nevermind, that was a mistake on my side.